### PR TITLE
Add hair pulato

### PR DESCRIPTION
**Pull request by DreamlyJack**
## About the PR

Add new hair "Pulato"
## Why / Balance

No balance
## Technical details

Just add hair pulato
## Media

lolkek
lolkek1

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: DreamlyJack
- add: Added Hair Pulato

### CODE REVIEW

**Comment by Djungelskog2** [comment]
oh my gah makima

File: Resources/Textures/Mobs/Customization/human_hair.rsi/meta.json [hunk 1]
```diff
@@ -567,6 +567,10 @@
         "name": "protagonist",
         "directions": 4
       },
+      {
+        "name": "pulato",
```
Comment (slarticodefast) [inline Resources/Textures/Mobs/Customization/human_hair.rsi/meta.json:571]: Please add an attribution to the `copyright` line at the top of the file.
For example "pulato.png made by DreamlyJack for SS14". If you did not make it yourself or made it by modifying an existing sprite, then make sure to mention that.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Textures/Mobs/Customization/human_hair.rsi/meta.json (hunk 1 above)
Comment (DreamlyJack) [reply Resources/Textures/Mobs/Customization/human_hair.rsi/meta.json:571]: ok
thanks

**Comment by NoElkaTheGod** [comment]
Nice

**Comment by FairlySadPanda** [comment]
I read this as "add hair pukeko" and tbh this hairdo is totally pukeko-pilled

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**